tests/pretrained_test.py

```diff
@@ -407,3 +408,7 @@ def test_transformer_qa(self):
     def test_pretrained_models(self, model_id, model_card):
         # Each model in pretrained_models should have an archive and registered_predictor_name.
         assert model_card.archive_file is not None
+
+    @pytest.mark.parametrize("task_id, task_card", get_tasks().items())
```

Comment:
I think we should also have a test that ensures the `task_id` for each model card is valid.